scene/gui/code_edit.cpp

```diff
@@ -436,12 +436,12 @@ void CodeEdit::gui_input(const Ref<InputEvent> &p_gui_input) {
 	}
 
 	/* If a modifier has been pressed, and nothing else, return. */
-	if (!k->is_pressed() || k->get_keycode() == Key::CTRL || k->get_keycode() == Key::ALT || k->get_keycode() == Key::SHIFT || k->get_keycode() == Key::META) {
+	if (!k->is_pressed() || k->get_keycode() == Key::CTRL || k->get_keycode() == Key::ALT || k->get_keycode() == Key::SHIFT || k->get_keycode() == Key::META || k->get_keycode() == Key::CAPSLOCK) {
 		return;
 	}
 
-	/* Allow unicode handling if:              */
-	/* No Modifiers are pressed (except shift) */
+	/* Allow unicode handling if:							*/
+	/* No Modifiers are pressed (except shift and CapsLock)	*/
```

Comment:
```suggestion
	// Allow unicode handling if:\
	// * No Modifiers are pressed (except shift and CapsLock)
```

Reply:
The fact that Shift is not capitalized, but CapsLock is... is annoying. 👀

Reply:
now I have to modify -> commit --amend -> push --force ?